Recruit-Training/Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -29,4 +29,10 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
     const double initialStateOfChargePercent_;
 
     // Add your necessary variables here
+    double current_;
+    double oldCurrent_;
+    QTime newTime_;
+    QTime oldTime_;
+    double currentAmphoursUsed_ = 0;
```

Comment:
Be consistent when initializing your variables, usually initialize within the constructor. currentAmpHoursUsed_ should be initialized withing the constructor